lib/kitchen/document.rb

```diff
@@ -18,6 +20,7 @@ class Document
     #   @return [Selectors::Base]
     def_delegators :config, :selectors
 
+    # rubocop:disable Layout/LineLength
```

Comment:
You can add a regular expression to ignore this cop for lines with comments (in .rubocop.yml)

https://stackoverflow.com/questions/26342570/rubocop-linelength-how-to-ignore-lines-with-comments#comment81397139_42833040

```yaml
Layout/LineLength:
  Max: 80
  IgnoredPatterns: ['(\A|\s)#']
```

@kswanson33 just had this problem in [her PR](https://github.com/openstax/kitchen/pull/114/files#diff-65a4951dc80fa8de7805fc7491d956d6982228167804db2e3ff0ae87c011d204R27) where she had to do a funny line break for a comment with code.  So I think it'd be good to turn it off for comments.

Then you can get rid of these `disable` / `enable` directives.